ovos_audio/service.py

```diff
@@ -9,10 +11,11 @@
 from ovos_plugin_manager.g2p import get_g2p_lang_configs, get_g2p_supported_langs, get_g2p_module_configs
 from ovos_plugin_manager.tts import TTS
 from ovos_plugin_manager.tts import get_tts_supported_langs, get_tts_lang_configs, get_tts_module_configs
+from ovos_utils.file_utils import resolve_resource_file
 from ovos_utils.log import LOG
 from ovos_utils.metrics import Stopwatch
 from ovos_utils.process_utils import ProcessStatus, StatusCallbackMap
-from ovos_utils.signal import check_for_signal
```

Comment:
This should be kept with a deprecation wrapper for at least one version (I think this is responsible for failing tests

Reply:
nothing is creating the signals anymore.... the tests were failing before this change too